modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/BSTValidator.java

```diff
@@ -75,12 +76,11 @@ public Credential validate(Credential credential, RequestData requestData) throw
     public boolean authenticate(String binarySecurityToken) throws
```

Comment:
this method only seems to be used by another method within the class itself. Better setting access to this as "private" then.